cmd/copy.go

```diff
@@ -83,6 +83,7 @@ type rawCopyCmdArgs struct {
 	// filters from flags
 	listOfFilesToCopy string
 	recursive         bool
+	copyContents      bool
```

Comment:
>copyContents [](start = 1, length = 12)

rename to correspond to the flag name.

This whole command "copies contents" so the field name can be quite confusing.